### DISCUSSION TITLE
v1.29.1 -> .2 Update not working.

### DISCUSSION THREAD

**Discussion by njordan77** [Koenkk/zigbee2mqtt#16156] [General]
Hi,
i'm on RPI4 with a v1.29.1 trying to update from command line via sudo ./update. but here the first time after many updates i have an error and not sure if its my installation,... Tried reboot,..... Any hint what is the problem?

Thanks

```
Updating 7d67ffc4..bb3e8f6d
error: Your local changes to the following files would be overwritten by merge:
        package-lock.json
Please commit your changes or stash them before you merge.
Aborting
```

**Comment by Koenkk** [comment]
Execute `git checkout package-lock.json`
